HIP/hip-0000-free-network-info-query.md

```diff
@@ -0,0 +1,63 @@
+| hip               | title              | author         | type            | status         | created       | discussions |
+| ----------- | -----------  | ----------- | ----------- |----------- | ----------- | ----------- |
+| TBD            | Free network info query | Simi Hunjan (@SimiHunjan ) |Service| Draft | 2021-06-09   |   https://github.com/hashgraph/hedera-improvement-proposal/discussions/82
```

Comment:
Do you mind renaming the file to `hip-20.md` and adding `hip-20` in the header table?

Reply:
updated to hip 21